app/controllers/api/v3/channels_controller.rb

```diff
@@ -30,7 +30,8 @@ def subscribe
   def unsubscribe
     @mammoth = Mammoth::Channels.new
     @user = @mammoth.unsubscribe(channel_id_param, acct_param)
-    UpdateForYouWorker.perform_async({ acct: acct_param, rebuild: true })
+    # Set Queue specificlly for 'rebuild'
+    UpdateForYouWorker.set(queue: 'mammoth_critial').perform_async({ acct: acct_param, rebuild: true })
     render json: @user
```

Comment:
we now have a dedicated queue for 'rebuilds' and other critical tasks for mammoth. Since the worker is the same logic it's default queue is `mammoth_default`, but when passing the `rebuild: true` we manually set the Queue to critical.